tapir/utils/management/commands/generate_test_data_functions.py

```diff
@@ -555,6 +560,81 @@ def generate_purchase_baskets():
     PurchaseBasket.objects.bulk_create(baskets)
 
 
+def generate_credit_account():
```

Comment:
It's nice that you added test data.
Maybe do it that not every member has credit, many members don't use that system.